server/game/cards/11.4-MoD/GreyGhost.js

```diff
@@ -0,0 +1,63 @@
+const DrawCard = require('../../drawcard');
+
+class GreyGhost extends DrawCard {
+    constructor(owner, cardData) {
+        super(owner, cardData);
+
+        this.tracker = new ShadowsTracker(this.game, this);
+    }
+
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Choose character(s)',
+            cost: ability.costs.kneelSelf(),
+            handler: () => {
+                this.game.promptForSelect(this.controller, {
```

Comment:
This should be done via the target API (which makes sure that 1) the ability can't be initiated if there are no valid targets, and 2) that targets are chosen and communicated to the opponent before they decide whether to cancel).